packages/block-library/src/post-featured-image/block.json

```diff
@@ -58,6 +58,13 @@
 		}
 	},
 	"usesContext": [ "postId", "postType", "queryId" ],
+	"example": {
+		"viewportWidth": 350,
+		"attributes": {
+			"width": 300,
+			"height": 150
+		}
```

Comment:
```suggestion
	
```
We might not need that when https://github.com/WordPress/gutenberg/pull/63177 lands